css/jquery.multiselect.css

```diff
@@ -19,7 +19,7 @@
 .ui-multiselect-checkboxes li { clear:both; font-size:0.9em; list-style: none; padding-right:3px; }
 .ui-multiselect-checkboxes > .ui-multiselect-optgroup { padding: 3px; }
 .ui-multiselect-columns { display: inline-block; vertical-align: top; }
-.ui-multiselect-checkboxes > .ui-multiselect-optgroup > a { border-bottom:1px solid; cursor: pointer; display:block; font-weight:bold; margin:1px 0; padding:3px; text-align:center; text-decoration:none; }
+.ui-multiselect-checkboxes > .ui-multiselect-optgroup > li > a { border-bottom:1px solid; cursor: pointer; display:block; font-weight:bold; margin:1px 0; padding:3px; text-align:center; text-decoration:none; }
```

Comment:
Per [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul), if the list is properly nested. the optgroup class name would be on an li in the top level list, and the anchor tag would also land in that li tag.  Therefore, I think if the list is properly nested, then no change is needed here.

Reply:
The anchor tag is under an `li` which is the only valid descendant of a `ul`. This fixes the issue of an `a` tag being nested under a `ul`.

What are you hoping to have changed?